### DISCUSSION TITLE
positive NPV values

### DISCUSSION THREAD

**Discussion by HuihuiYangshell** [NatLabRockies/GEOPHIRES-X#224] [Q&A] [answered]
How could I tune the results to give positive NPV values? any suggestion about the key parameters?

**Comment by kfbeckers** [comment, marked as answer]
Increasing the NPV can be obtained by either decreasing your cost and/or increasing your revenue. Often the wellfield costs are the biggest expense so you can try decreasing your drilling cost. Increasing your revenue can be achieved by, e.g., considering a higher electricity price. You can also try playing with financial incentives (e.g., the ITC/PTC) or assuming a lower discount rate.

**Reply by softwareengineerprogrammer** [reply]
(Note [my answer](https://github.com/NREL/GEOPHIRES-X/discussions/224#discussioncomment-9513220) is complementary this one)

**Comment by softwareengineerprogrammer** [comment]
@HuihuiYangshell: @kfbeckers discussed this briefly today - of course, it is a complicated question, but in terms of parameters that have significant impact on NPV and can be intuitively adjusted, here are a few ideas to get you started:

* Drilling cost has biggest expense - see `Well Drilling Cost Correlation` in https://nrel.github.io/GEOPHIRES-X/parameters.html#economics 
* Revenue is dependent on electricity price - `Starting Electricity Sale Price`, `Ending Electricity Sale Price`, `Electricity Escalation Start Year`, `Electricity Escalation Rate Per Year` are relevant
* PTC/ITC - `Production Tax Credit Electricity` and related, `Investment Tax Credit Rate` and related
